.github/workflows/perf_annotate.py

```diff
@@ -0,0 +1,64 @@
+import argparse
+import json
+from os import environ
+from pathlib import Path
+from typing import List, Any, Dict
+
+
+class Args(argparse.Namespace):
+    json_file: Path
+
+
+def main():
+    arg_parser = argparse.ArgumentParser()
+    arg_parser.add_argument("json_file", type=Path)
+    arg_parser.parse_args(namespace=Args)
+
+    bench_json = Args.json_file.read_text(encoding="utf-8")
```

Comment:
> This will not work because json_file is an instance attribute, not a class attribute.

It actually does work _because_ `json_file` _is_ a class attribute. Instance attributes in python involve a `self` scope.